android/pytorch_android/CMakeLists.txt

```diff
@@ -155,3 +158,12 @@ if(USE_VULKAN)
 endif()
 
 target_link_libraries(pytorch_jni ${pytorch_jni_LIBS})
+
+install(TARGETS pytorch_jni
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}) #For windows
+
+if(MSVC)
+  install(TARGETS pytorch_jni DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
One thing you are missing is that you forget to install the PDB fles on debug builds. See https://github.com/pytorch/FBGEMM/blob/master/CMakeLists.txt#L253-L255